fastrtps_cmake_module/cmake/Modules/FindFastRTPS.cmake

```diff
@@ -70,3 +70,11 @@ find_package_handle_standard_args(FastRTPS
   REQUIRED_VARS
     FastRTPS_INCLUDE_DIR
 )
+
+find_library(FastRTPS_LIBRARY
+    NAMES fastrtps)
+
+find_library(FastCDR_LIBRARY
+    NAMES fastcdr)
+
+set(FastRTPS_LIBRARIES ${FastRTPS_LIBRARY} ${FastCDR_LIBRARY})
```

Comment:
Fixed in https://github.com/ros2/rmw_fastrtps/pull/75/commits/a58e74632a9fc2863217eb8664d713cf66ae82bf